Makefile

```diff
@@ -11,9 +11,23 @@ export BUILD_INFO := $(shell ./build-tools/version-tool build-info)
 
 GO_BUILD_FLAGS=-v -ldflags "-extldflags \"-static\" -X main.version=$(BUILD_VERSION) -X main.buildInfo=$(BUILD_INFO)"
 
-# Allow users to pass in BASE_OS build options (alpine or rhel7)
+# Allow users to pass in BASE_OS build options (alpine or rhel7) 
 BASE_OS ?= alpine
 
+# This is for builds not triggered through Travis CI 
+ifndef LICENSE_STRICT
+	LICENSE_STRICT=false
+endif
```

Comment:
I think there is a simpler way to do this. `LICENSE_STRICT ?= false` should set the variable to false if it isn't already set.

Reply:
Per our discussion, if testing is taking too long to verify all scenarios, I will defer it for now.